manim/_config/utils.py

```diff
@@ -20,18 +20,24 @@
 import re
 import sys
 from collections.abc import Mapping, MutableMapping
-from enum import EnumMeta
 from pathlib import Path
-from typing import Any, ClassVar, Iterable, Iterator, NoReturn
+from typing import TYPE_CHECKING, Any, ClassVar, Iterable, Iterator, NoReturn
 
 import numpy as np
-from typing_extensions import Self
 
-from .. import constants
-from ..constants import RendererType
-from ..typing import StrPath, Vector3
-from ..utils.color import ManimColor
-from ..utils.tex import TexTemplate, TexTemplateFromFile
+from manim import constants
+from manim.constants import RendererType
+from manim.utils.color import ManimColor
+from manim.utils.tex import TexTemplate, TexTemplateFromFile
+
+if TYPE_CHECKING:
+    from enum import EnumMeta
+
```

Comment:
```suggestion
```
To keep spacing consistent.

Reply:
I agree, but that change was forced by `isort` in `pre-commit`. I tried to remove them again now, but `isort` inserted them once again. :man_shrugging: